### DISCUSSION TITLE
一些想法不吐不快

### DISCUSSION THREAD

**Discussion by shadowlgd** [WeNeedHome/SummaryOfLoanSuspension#454] [General]
唐山，郑州710，这次停贷，等等，有了一些想法。
1，尊重每个人的发言权。你可以反对他，反驳他甚至骂他，但是不能不让他说话。这点在710感受颇深，我是个老 装逼乎er，一向是看不上微博的，认为那就只是娱乐新闻APP，在储户第一次郑州游行我评论后直接把我账户永删后我依然这么想。但是710我除了能在微博上看到一点信息，其他地方都看不到。
2，能救人民的只有人民自己。包青天不行，法治也不行，只有人民有权利，能对不合理不公平的事说不，而不是只能对不合法，不合利的说不。

**Comment by sqpk** [comment]
关键你说不了不，嘴都缝上了

**Comment by JACWATA** [comment]
反对无效，下楼做核酸

**Reply by SiminePu** [reply]
秀儿

**Comment by xiaer93** [comment]
公民有言论自由，现在红线很多，都不知道哪里是红线。

**Reply by blacklion-spec** [reply]
威权体制嘛，上面的看法就是法律，寻衅滋事罪可以治一切上面看不惯的人

**Comment by trunklater** [comment]
反对无效，开门检查天然气的。

**Comment by Colunmdear** [comment]
想啥言论自由呢

**Comment by yuyeshun** [comment]
头疼医嘴，脚疼医嘴

**Comment by dalezhang93** [comment]
自由是靠争取来的，还得感谢唐山拍视频的人，江西老表不服就干的勇气，打响第一抢。其实这样的环境中长大，不知不觉思想都被阉割了

**Comment by xiaer93** [comment]
宪法赋予大家言论自由的权利，只能在github这种网站表达~~~真是时代的悲哀。

**Reply by ruo-xi** [reply]
宪法还说人民民主专政呢，唉。曾经还是无产阶级专政，灵活的宪法 不知道被谁给改成人民民主了？

**Comment by k0spider** [comment]
陈吴初期都得有个人是兵长，光靠老百姓没枪没炮的，谁会听？

**Comment by BlackBerryPi** [comment]
开门，白衬衫送温暖！

**Comment by AlphaXu917** [comment]
工农阶级铸造的利剑，最终挥向了他们的子孙后代、、、心寒

**Reply by delfws** [reply]
至少没拿一针一线

**Comment by liuwenwu520** [comment]
中国真的没有言论自由，小说，抖音，微博，很多都有红区，看着美国还可以调侃总统，这真是一个最糟糕的时代

**Reply by ppwq2022** [reply]
欧美自由平等民主，黑铁拳砸向你的时候不知道你还会不会自由，底层到哪都是底层，西方民主被洗脑更彻底，还搁那说可笑的自由

**Reply by liuwenwu520** [reply]
我只是说相比之下更不自由，因为现在的业主连发生的平台都找不到，抖音、微博还有很多新闻都在删除，凭什么不让他们发声

**Reply by liuwenwu520** [reply]
还有烂尾楼业主群被强制解散，这真是自由吗，我们为什么要和欧美比较，我只说中国的情况

**Reply by Galthen** [reply]
> 欧美自由平等民主，黑铁拳砸向你的时候不知道你还会不会自由，底层到哪都是底层，西方民主被洗脑更彻底，还搁那说可笑的自由

国内国外都在洗脑，都不是什么好人，但是在言论方面，国外管控的比国内松散很多。
这么说，你可能更容易接受点。
理智的分析每个国家的优缺点就行。

**Reply by taopeng-life** [reply]
> 还有烂尾楼业主群被强制解散，这真是自由吗，我们为什么要和欧美比较，我只说中国的情况

是的  抖音、微博 小红书 全是审查 根本不能发声，全是删除

**Reply by ppwq2022** [reply]
> 这么说，你可能



> 

你这说的就更可笑了，言论自由？ 川普帐号被封解释下？  所谓的自由不过是上层精英放松你的链子，搁这清高呢，你问昂撒人让你自由了吗

**Reply by liuwenwu520** [reply]
现在是业主在中国的各个平台上面不能够进行发声，业主们抱团取暖的地方都没有，我们中国是否能够出一个发声平台给这些弱势群体，让政府、全体中国人听到他们的声音，而不是在 github 这个还具有门槛的网站上面发声

**Reply by whyqiuqiu** [reply]
西方体制也终究是空中楼阁 不能以偏概全

**Reply by Galthen** [reply]
> > 这么说，你可能
> 
> > 
> 
> 你这说的就更可笑了，言论自由？ 川普帐号被封解释下？ 所谓的自由不过是上层精英放松你的链子，搁这清高呢，你问昂撒人让你自由了吗

你能理智和仔细的看下我的回复再说吗？你认为我哪句话说的不对。

**Reply by ppwq2022** [reply]
> > > 这么说，你可能
> > 
> > 
> > > 
> > 
> > 
> > 你这说的就更可笑了，言论自由？ 川普帐号被封解释下？ 所谓的自由不过是上层精英放松你的链子，搁这清高呢，你问昂撒人让你自由了吗
> 
> 你能理智和仔细的看下我的回复再说吗？你认为我哪句话说的不对。

理中客又来了？反思券不多了

**Reply by Galthen** [reply]
> > > > 这么说，你可能
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > 你这说的就更可笑了，言论自由？ 川普帐号被封解释下？ 所谓的自由不过是上层精英放松你的链子，搁这清高呢，你问昂撒人让你自由了吗
> > 
> > 
> > 你能理智和仔细的看下我的回复再说吗？你认为我哪句话说的不对。
> 
> 理中客又来了？反思券不多了

我认为你在胡搅蛮缠，而不是就事论事，跟你讨论没有意义，不再回复了，再见。

**Reply by flame618** [reply]
> > 这么说，你可能
> 
> > 
> 
> 你这说的就更可笑了，言论自由？ 川普帐号被封解释下？ 所谓的自由不过是上层精英放松你的链子，搁这清高呢，你问昂撒人让你自由了吗

一提到言论自由你们这些人就拿川普被封说事，二极管吗？没人说美国是绝对言论自由，但是比中国好个十万八千里，美国也不是世界上言论最自由的国家，最自由的是挪威。有个新闻自由指数，可以体现言论自由，180个国家，中国排倒数第5，河南白衣打人事件有一个国内媒体报道吗？

**Reply by blacklion-spec** [reply]
> > 这么说，你可能
> 
> 
> 
> 
> 
> 
> 
> > 
> 
> 
> 
> 你这说的就更可笑了，言论自由？ 川普帐号被封解释下？  所谓的自由不过是上层精英放松你的链子，搁这清高呢，你问昂撒人让你自由了吗

你理解言论自由吗？言论自由是公权力无权干涉人民的言论，要是言论自由是私权对私权的话，我天天骂你娘，你也不能拿我怎么样啰？川普被封锁，是被一家私有企业封锁，本来就和言论自由定义不冲突，从某种角度说，这是私权限制公权。所以你得好好理解什么是言论自由。

**Comment by shawn-yee** [comment]
反对无效，封你账号

**Comment by sducxh** [comment]
说的没错，只能自救，但是...几个人的力量很难，甚至几个人都聚合不起来~ 我山东宇宙中心房子7年了，没房产证，开发商代收了税款，没有发票，求助12345，下发到房管局和税务，活稀泥。。哎~烂了的国

**Comment by chizi-chizi** [comment]
只有跳到笼子外， 才能看到笼子里的黑暗。

**Comment by dongyc519** [comment]
这里没有自由的人民，只有自由的韭菜。

**Reply by purpose1113** [reply]
agree

**Reply by chenhaizano** [reply]
赞同

**Comment by Galthen** [comment]
“愿中国青年都摆脱冷气,只是向上走,不必听自暴自弃者流的话。能做事的做事,能发声的发声。有一分热,发一分光。就令萤火一般,也可以在黑暗里发一点光,不必等候炬火。”

**Reply by purpose1113** [reply]
就差当代鲁迅了

**Reply by iosyyy** [reply]
这句话后面乃句最出名
> 此后如竟没有炬火,我便是唯一的光。

**Reply by Galthen** [reply]
> 就差当代鲁迅了

我觉得这句话很实用。业主们骂来骂去解决不了问题，强制停贷就是在前进，我们的发声就是对他们的支持，有其他业主跟随停贷也是支持。
为了让生活更好，做自己力所能力的小事。无论是国外还是国外，谁阻拦大家追求美好生活的，都是敌人。

**Reply by 710baichenshan** [reply]
> 就差当代鲁迅了

鲁迅要是在当代早就被抓起来了。

**Reply by teddycode** [reply]
当代鲁迅起不来了，给了点利益就闭嘴了，敢说是绝大多数人，这种金钱观就是从小被灌到大的，真的蝇头小利就会让他们土崩瓦解。

**Comment by AlphaXu917** [comment]
需要人呐喊，不能沉默。
 
发件人： Galthen
发送时间： 2022-07-15 16:53
收件人： WeNeedHome/SummaryOfLoanSuspension
抄送： Alpha Xu; Comment
主题： Re: [WeNeedHome/SummaryOfLoanSuspension] 一些想法不吐不快 (Discussion #454)
就差当代鲁迅了
我觉得这句话很实用。业主们骂来骂去解决不了问题，强制停贷就是在前进，我们的发声就是对他们的支持，有其他业主跟随停贷也是支持。
为了让生活更好，做自己力所能力的小事。无论是国外还是国外，谁阻拦大家追求美好生活的，都是敌人。

**Reply by randufeng** [reply]
当代鲁迅可能也会被一个红码,一个又一个的删号封停变成这个社会的透明人,生于此,何其哀

**Comment by AlphaXu917** [comment]
不可能再有鲁迅了

发件人： randufeng
发送时间： 2022-07-15 16:59
收件人： WeNeedHome/SummaryOfLoanSuspension
抄送： Alpha Xu; Comment
主题： Re: [WeNeedHome/SummaryOfLoanSuspension] 一些想法不吐不快 (Discussion #454)
当代鲁迅可能也会被一个红码,一个又一个的删号封停变成这个社会的透明人,生于此,何其哀

**Comment by ethenlong** [comment]
我只感觉到快，加速加速加速！！！

**Comment by Crazy-Creeper** [comment]
现在每一个披露信息的人都要承担巨大的风险。

**Comment by z-huabao** [comment]
> 唐山，郑州710，这次停贷，等等，有了一些想法。 1，尊重每个人的发言权。你可以反对他，反驳他甚至骂他，但是不能不让他说话。这点在710感受颇深，我是个老 装逼乎er，一向是看不上微博的，认为那就只是娱乐新闻APP，在储户第一次郑州游行我评论后直接把我账户永删后我依然这么想。但是710我除了能在微博上看到一点信息，其他地方都看不到。 2，能救人民的只有人民自己。包青天不行，法治也不行，只有人民有权利，能对不合理不公平的事说不，而不是只能对不合法，不合利的说不。

[苦笑]跟我一样，本来也不看微博，但是710知乎上完全搜不到，反而微博还有
